examples/update-autotask-env-variables/code/index.js

```diff
@@ -0,0 +1,8 @@
+const deps = require('./deps');
+
+exports.handler = async function () {
```

Comment:
Not sure i understand what is this example demonstrating

Reply:
Good catch, we don't need this anymore.